package.json

```diff
@@ -67,22 +67,16 @@
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
-    "aead-js": "^0.1.0",
+    "bcrypto": "5.1.0",
     "buffer": "^5.4.3",
     "debug": "^4.1.1",
-    "futoin-hkdf": "^1.3.2",
-    "hash.js": "^1.1.7",
     "it-buffer": "^0.1.1",
     "it-length-prefixed": "^3.0.0",
     "it-pair": "^1.0.0",
     "it-pb-rpc": "^0.1.8",
     "it-pipe": "^1.1.0",
-    "libp2p-crypto": "^0.17.6",
+    "libp2p-crypto": "^0.17.1",
```

Comment:
why not latest? available

Reply:
idk, this was not on purpose. I'll update